src/node/markdownToVue.ts

```diff
@@ -21,6 +21,10 @@ export interface MarkdownCompileResult {
   includes: string[]
 }
 
+export function clearCache() {
+  cache.reset()
```

Comment:
**TODO:** change this to `cache.clear()` in vite-3 branch. `.reset()` is deprecated in newer versions of `lru-cache`.